tests/cli/test_init.py

```diff
@@ -195,47 +209,66 @@ def test_init_on_cloned_repo(isolated_runner, data_repository):
 
 
 @pytest.mark.integration
-def test_init_remote(isolated_runner):
+def test_init_remote(isolated_runner, project_init):
     """Test project initialization from a remote template."""
+    template, data, commands = project_init
+
     # create the project
-    new_project = Path("test-new-project")
+    new_project = Path(data["test_project"])
     assert not new_project.exists()
-    result = isolated_runner.invoke(cli, INIT + INIT_REMOTE, INPUT)
+    result = isolated_runner.invoke(
+        cli, commands["init_test"] + commands["id"] + commands["force"], commands["confirm"]
+    )
     assert 0 == result.exit_code
     assert new_project.exists()
     assert (new_project / ".renku").exists()
     assert (new_project / ".renku" / "renku.ini").exists()
     assert (new_project / ".renku" / "metadata.yml").exists()
 
 
-def test_init_with_parameters(isolated_runner):
+def test_init_with_parameters(isolated_runner, project_init):
     """Test project initialization using custom metadata."""
+    template, data, commands = project_init
+
     # create the project
-    new_project = Path("test-new-project")
+    new_project = Path(data["test_project"])
     assert not new_project.exists()
-    result = isolated_runner.invoke(cli, INIT + PARAMETERS_CORRECT + PARAMETER_NO_EQUAL)
+    result = isolated_runner.invoke(
+        cli, commands["init_test"] + commands["id"] + commands["parameters"] + commands["parameters_equal_missing"]
+    )
     assert 0 != result.exit_code
-    assert "Error: Invalid parameter value for --parameter " f'"{PARAMETER_NO_EQUAL[1]}"' in result.output
-
-    result = isolated_runner.invoke(cli, INIT + PARAMETERS_CORRECT + PARAMETER_EARLY_EQUAL)
+    assert (
+        "Error: Invalid parameter value for --parameter "
+        f'"{ commands["parameters_equal_missing"][1]}"' in result.output
+    )
+
+    result = isolated_runner.invoke(
+        cli, commands["init_test"] + commands["id"] + commands["parameters"] + commands["parameters_equal_early"]
+    )
     assert 0 != result.exit_code
-    assert "Error: Invalid parameter value for --parameter " f'"{PARAMETER_EARLY_EQUAL[1]}"' in result.output
+    assert (
+        "Error: Invalid parameter value for --parameter " f'"{commands["parameters_equal_early"][1]}"' in result.output
```

Comment:
Join the two strings to be a single f-string

Reply:
done, also in another occurence a couple of lines above